supabase/models/messaging/startNewConversation.ts

```diff
@@ -62,13 +66,31 @@ export default async function startNewConversation(
         'This is the start of your conversation.'
       );
 
-      return conversationId;
+      const { data: newUserConversation, error: newUserConversationError } =
+        await supabase
+          .from('user_conversations')
+          .select('*')
+          .eq('user_id', userID)
+          .eq('partner_id', donorID)
+          .eq('item_id', itemID)
+          .single();
+
+      if (newUserConversationError) {
+        console.error(
+          'Error fetching new user conversation:',
+          newUserConversationError
```

Comment:
instead of just console logging this can we `throw error` instead ?

That way we can grab that error on the frontend and decide there how to communicate to the user